File: linear_algebra/C++/UNK_BT10.cpp
#include <iostream>
using namespace std;

int main()
{
    double A[2][2];
    cout << "Nhap ma tran A:" << endl;
    for (int i = 0; i < 2; i++) {
        for (int j = 0; j < 2; j++) {
            cout << "A[" << i << "][" << j << "] = ";
            cin >> A[i][j];
        }
        cout<<endl;
    }
    double det = A[0][0]*A[1][1] - A[0][1]*A[1][0];
    cout<<"Det cua ma tran A la: "<<det<<endl;
    return 0;
}

File: linear_algebra/Python/UNK_BT10.py
A=[[0, 0],[0, 0]]
print('Nhap ma tran A:')
for i in range (2):
    for j in range (2):
        A[i][j]=float(input(f"A[{i}][{j}] = "))   
det=A[0][0]*A[1][1]-A[0][1]*A[1][0]
print('Det A =',det)
